Source/Core/Common/Concepts.h

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// This feature detection be removed once Dolphin Emulator infrastructure finishes upgrading
+#ifndef __cpp_concepts
+#error "C++20 concepts support not detected!  Update your compiler!"
+#endif
```

Comment:
Typo; "detection be removed" should be "detection should be removed". Though this `#error` doesn't seem to be triggering anymore, so I think you can just remove it?